content/docs/1.8.0/important-notes/_index.md

```diff
@@ -63,16 +64,33 @@ Longhorn v1.8.0 and later versions support usage of V2 volumes in Talos Linux cl
 
 ## Backup
 
+### Multiple Backupstores Support
+
+Starting with v1.8.0, Longhorn supports usage of multiple backupstores. You can configure backup targets to access backupstores on the **Setting/Backup Target** screen of the Longhorn UI. v1.8.0 improves on earlier Longhorn versions, which only allow you to use a single backup target for accessing a backupstore. Earlier versions also require you to configure the settings `backup-target`, `backup-target-credential-secret`, and `backupstore-poll-interval` for backup target management.
+
+> **IMPORTANT:**  
+> The settings `backup-target`, `backup-target-credential-secret`, and `backupstore-poll-interval` were removed from the global settings because backup targets can be configured on the **Setting/Backup Target** screen of the Longhorn UI. Longhorn also creates a default backup target (`default`) during installation and upgrades.
+
+Longhorn creates a default backup target (`default`) during installation and upgrades. The default backup target is used for the following:
+
+- System backups
+- Volumes that were created without a specific backup target name
+
+> **Tip:**
+> Set the default backup target before creating a new one.
+
+For more information, see [Setting a Backup Target](../snapshots-and-backups/backup-and-restore/set-backup-target), [Issue #5411](https://github.com/longhorn/longhorn/issues/5411) and [Issue #10089](https://github.com/longhorn/longhorn/issues/10089).
```

Comment:
Also mention how to configure the default backup store in the values.yaml or manifest

Reply:
or reference the page for configuring the default backup target.

Reply:
Added to reference the page for configuring the default backup target. Thanks.